po/de.po

```diff
@@ -131,36 +129,31 @@ msgstr ""
 "Spielen zu bieten."
 
 #: src/launchers/hgl.vala:7
-#, fuzzy
 msgid "Compatibility tools by Valve for running Windows software on Linux."
-msgstr "Kompatibilitätstool zum Ausführen von Windows-Spielen unter Linux"
+msgstr "Kompatibilitätstool zum Ausführen von Windows-Spielen unter Linux."
 
 #: src/launchers/hgl.vala:19 src/launchers/lutris.vala:15
-#, fuzzy
 msgid ""
 "Variant of Wine's VKD3D which aims to implement the full Direct3D 12 API on "
 "top of Vulkan."
 msgstr ""
 "Fork von Wine's VKD3D, der darauf abzielt, die volle Direct3D 12 API auf "
```

Comment:
Try to use another word than Fork here. We switched it to Variant to be clearer for newbies.